src/config/config.tsx

```diff
@@ -26,7 +26,7 @@ export const SYMBOL = SYMBOLS[DEFAULT_CHAIN_ID];
 export const EXCHANGE_ADDRESS: { [chainId: number]: string } = {
   [CHAIN_IDS.POLYGON_MAINNET]: '0x475b5834085bf61A3B330eBd1c689ff8459258C2',
   [CHAIN_IDS.POLYGON_TESTNET]: '0x0d4a11d5eeaac28ec3f61d100daf4d40471f1852',
-  [CHAIN_IDS.DAPPTOOLS]: '0x557767ffbe4244f01e2231728D568EF523Aa2121',
+  [CHAIN_IDS.DAPPTOOLS]: '0xBf4Bd3dCD8BF7BD34D70d44e3e1ABe97e2B3aE23',
```

Comment:
If this changes more often or different on each system, I suggest we can extract this in REACT environment variable.